src/main/java/aquarium/shop/basket/Basket.java

```diff
@@ -0,0 +1,61 @@
+package aquarium.shop.basket;
+
+import aquarium.shop.product.Product;
+import java.util.HashMap;
+
+
+public class Basket {
+
+    private static HashMap<Product,Integer> content = new HashMap<>();
```

Comment:
This structure is ok, however I would avoid to make it static. It's better to make Basker class some kind of singleton and create only one instance of it in application lifecycle.

baeldung.com/java-singleton